ed_perception/src/perception_plugin_image_recognition.cpp

```diff
@@ -31,7 +31,7 @@ namespace perception
 
 // ----------------------------------------------------------------------------------------------------
 
-PerceptionPluginImageRecognition::PerceptionPluginImageRecognition()
+PerceptionPluginImageRecognition::PerceptionPluginImageRecognition() : roi_margin_(0)
```

Comment:
Does this initialise roi margin to 0?
Preferably set it to 5, the default value

Reply:
In the initialize it is set to 5, before checking if it in tbe config.